app/api/services/models/user.py

```diff
@@ -0,0 +1,6 @@
+from .base import PatchParamsBase
+
+
+# Inherit from PatchParamsBase, to persist the original JSON input
+class UserPatchParams(PatchParamsBase):
```

Comment:
I decided not to add any fields to the `UserPatchParams` class, since we ignore them and just use the input JSON dict instead.